src/app/routes/article/getInitialData/index.ts

```diff
@@ -83,23 +83,14 @@ export default async ({
       ...(!isLocal && { agent, optHeaders }),
     });
 
-    const wsojURL = getRecommendationsUrl({
-      assetUri: pathname.replace(/(\.|\?).*/g, ''),
-      engine: 'unirecs_datalab',
-      engineVariant: '',
+    const wsojData = await getAdditionalPageData({
+      pageData: json,
+      service,
+      variant,
+      env,
+      path: pathname.replace(/(\.|\?).*/g, ''),
```

Comment:
What exactly is this regex doing?

Reply:
It's to remove .amp and env=render_live parameters from the path, since getAdditionalData needs just the 'service/article' portion of the path to build the correct URL for retrieving recommendations.

Reply:
Cool, could we make a const for the regex, something like ampRendererEnvRegex? Just for clarity if someone comes across it and doesn't know regex very well, like me haha